src/components/largePipelineWarning/index.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { toggleDisplayLargeGraph } from '../../actions/graph';
+import './largePipelineWarning.css';
+
+export const LargePipelineWarning = ({
+  nodeCount,
+  onToggleDisplayLargeGraph
+}) => {
+  return (
+    <div className="warning">
+      <div className="warning__title">Your pipeline is large.</div>
```

Comment:
@stichbury I know this has been discussed, but I think this title itself might be clearer just using the line that follows it ("Your pipeline might take a while") and then adjust the subtitle, so to avoid being unnecessarily prescriptive?

Reply:
Sounds good!

Reply:
This is a good suggestion, but on revisiting your suggested wording it sounds more like that the pipeline is loading in the background, which is not the case given the intent of this warning ( which is to prevent rendering a large graph by prompting the user to filter by tags/ nodes).

I would say the previous wording in the design ( and as discussed) it more clear.